.github/workflows/check-coverage.yml

```diff
@@ -67,6 +67,11 @@ jobs:
           echo "Running e2e tests"
           cd e2e
           go test -e2eMode="coverage" -timeout 60m -v -run Test_All ./
+      - name: Upload e2e artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: scenario-logs
+          path: e2e/scenario-logs
```

Comment:
something I was meaning to do on the ADO side - can we upload a new bundle that doesn't include scenario-logs/*/vmssId.txt? those text files are only there such that we can ensure the ephemeral VMSS are deleted (we have an 'ensure vmss deletion step in the ADO pipeline that reads the ids from those text files and does 'az delete --ids ...' on them

I'm also realizing now that we don't have the same sort of 'ensure vmss deletion' step on the GH action side...

we should really choose one and pivot towards that

Reply:
might also be worth leaving those text files lying around so we can use them to manually delete stuff via az cli if needed....

I'll leave it up to you

Reply:
(can also just go use the portal to delete... :D)

Reply:
I will be looking into how i could combine ADO running e2e and gh action taking results of that and handling everything else